src/30_Advanced/SeatMap.html

```diff
@@ -36,6 +36,7 @@
       :root {
       --abe-light-pink: rgba(182, 8, 69, 0.5);
       --abe-pink: rgb(182, 8, 69);
```

Comment:
Can you replace these with `--color-primary` and `--color-secondary`.

Reply:
done